src/env-inl.h

```diff
@@ -700,9 +700,8 @@ inline void Environment::remove_sub_worker_context(worker::Worker* context) {
   sub_worker_contexts_.erase(context);
 }
 
-inline bool Environment::is_stopping_worker() const {
-  CHECK(!is_main_thread());
-  return worker_context_->is_stopped();
+inline bool Environment::is_stopping() const {
+  return thread_stopper_.IsStopped();
```

Comment:
You may be able to drop the `node_worker.h` include now from this file?

Reply:
@addaleax - done, removed.